tests/examples/source/devfiles/nodejs/devfileV2.yaml

```diff
@@ -0,0 +1,35 @@
+schemaVersion: "2.0.0"
+metadata:
+  name: test-devfile
+projects:
+  - name: nodejs-web-app
+    git: 
+      location: "https://github.com/che-samples/web-nodejs-sample.git"
+components:
+  - container:
+      image: quay.io/eclipse/che-nodejs10-ubi:nightly
+      mountSources: true
+      name: "runtime"
+      memoryLimit: 1024Mi
+      env:
+      - name: FOO
+        value: "bar"
+      endpoints:
+        - name: '3000/tcp'
+          targetPort: 3000
+          # Not parsing currently, added to validate JSON Schema
```

Comment:
why are we not parsing `configuration`?

Reply:
Need to update the cooment, We are parsing but not using the values.